extra/1-radio-stations.js

```diff
@@ -7,24 +7,53 @@
 /**
  * First, let's create a function that creates a list of all the frequencies.
  * Call this function `getAllFrequencies`.
+ * 
  *
  * This function should:
  * - Create an array  starting at 87 and ending in 108
  * - Should return this array to use in other functions
  */
 
+
 // `getAllFrequencies` goes here
 
+
+function getAllFrequencies() {
+  
```

Comment:
There are few blank lines here that can be deleted.
```suggestion
    let frequencies = [];
  for (let frequency = 87; frequency <= 108; frequency++) {
    frequencies.push(frequency);
  }
  return frequencies;
}
```

Reply:
:+1: